cvat/settings/base.py

```diff
@@ -769,3 +769,15 @@ class CVAT_QUEUES(Enum):
 
 # Indicates the maximum number of days a file or directory is retained in the temporary directory
 TMP_FILE_OR_DIR_RETENTION_DAYS = 3
+
+ABOUT_INFO = {
+    "name": "Computer Vision Annotation Tool",
+    "title": "Open Data Annotation Platform",
```

Comment:
I think calling this a "title" risks confusion with the "name" field. Maybe it should be called something like "subtitle", or "tagline"?